### PR TITLE
Update landscape.yml

### PR DESCRIPTION
**Pull request by mrsjackson76**
Adding CNTi to LFN

### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [x] Is your project closed source or, if it is open source, does your project have at least 250 GitHub stars?
* [x] Have you picked the single best (existing) category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Have you included a URL for your SVG or added it to `hosted_logos` and referenced it there?
* [x] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [x] ~10 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

**Comment by lfn-github** [comment]
<pre>Build failed because of:
bad indentation of a mapping entry (26:15)

 23 |               - repo_url: https://github.com/ ...
 24 |               - repo_url: https://github.com/ ...
 25 |               - repo_url: https://github.com/ ...
 26 |               name: CNTi
--------------------^
 27 |             allow_duplicate_repo: true
 28 |             homepage_url: https://lfnetworkin ...</pre>

**Comment by lfn-github** [comment]
<pre>Build failed because of:
bad indentation of a mapping entry (26:15)

 23 |               - repo_url: https://github.com/ ...
 24 |               - repo_url: https://github.com/ ...
 25 |               - repo_url: https://github.com/ ...
 26 |               name: CNTi
--------------------^
 27 |             allow_duplicate_repo: true
 28 |             homepage_url: https://lfnetworkin ...</pre>

**Comment by lfn-github** [comment]
<pre>Build failed because of:
bad indentation of a mapping entry (510:10)

 507 |             description: &gt;
 508 |               The Nephio project is buil ...
 509 |               as 5G Network Functions, a ...
 510 |          - item:
----------------^
 511 |           name: Paraglider
 512 |             allow_duplicate_repo: true</pre>

**Comment by lfn-github** [comment]
<pre>Build failed because of:
bad indentation of a mapping entry (510:10)

 507 |             description: &gt;
 508 |               The Nephio project is buil ...
 509 |               as 5G Network Functions, a ...
 510 |          - item:
----------------^
 511 |            name: Paraglider
 512 |             allow_duplicate_repo: true</pre>

File: landscape.yml [hunk 1]
```diff
@@ -515,6 +507,16 @@ landscape:
             description: >
               The Nephio project is building a Kubernetes-based automation platform for deploying and managing highly distributed, interconnected workloads such
               as 5G Network Functions, and the underlying infrastructure on which those workloads depend.
+         - item:
+           name: Paraglider
```
Comment (bramwelt) [inline landscape.yml:511]:
```suggestion
          - item:
            name: Paraglider
```

File: landscape.yml [hunk 2]
```diff
@@ -515,6 +507,16 @@ landscape:
...
+            allow_duplicate_repo: true
+            homepage_url: https://paragliderproject.io
+            logo: logos/paraglider-logo.svg
+            twitter: https://twitter.com/LF_Networking
+            crunchbase: https://www.crunchbase.com/organization/paraglider
+            repo_url: https://github.com/paraglider-project/paraglider
+           description: |
+           Paraglider is a cross-cloud control plane for configuring cloud networks.
```
Comment (bramwelt) [inline landscape.yml:519]:
```suggestion
            description: |
            Paraglider is a cross-cloud control plane for configuring cloud networks.
```

**Review by bramwelt — CHANGES_REQUESTED** [review]

**Comment by lfn-github** [comment]
<pre>Build failed because of:
bad indentation of a sequence entry (510:9)

 507 |             description: &gt;
 508 |               The Nephio project is bui ...
 509 |               as 5G Network Functions,  ...
 510 |         - item:
---------------^
 511 |           name: Paraglider
 512 |             allow_duplicate_repo: true</pre>

**Comment by lfn-github** [comment]
<pre>Build failed because of:
bad indentation of a mapping entry (510:10)

 507 |             description: &gt;
 508 |               The Nephio project is buil ...
 509 |               as 5G Network Functions, a ...
 510 |          - item:
----------------^
 511 |            name: Paraglider
 512 |            allow_duplicate_repo: true</pre>

**Comment by lfn-github** [comment]
<pre>Build failed because of:
No cached entry, and Paraglider has issues with logo: logos/paraglider-logo.svg, ENOENT: no such file or directory, open &#39;/opt/build/repo/hosted_logos/logos/paraglider-logo.svg&#39;
No cached entry, and CNTi has issues with logo: logos/CNTi_logo_COLOR.svg, ENOENT: no such file or directory, open &#39;/opt/build/repo/hosted_logos/logos/CNTi_logo_COLOR.svg&#39;
Item CNTi has no image_data
Item Paraglider has no image_data</pre>

File: landscape.yml [hunk 3]
```diff
@@ -23,31 +23,23 @@ landscape:
               - repo_url: https://github.com/opnfv/functest-xtesting
               - repo_url: https://github.com/opnfv/cirv-sdv
               - repo_url: https://github.com/opnfv/barometer
+          - item:
+            name: CNTi
+            allow_duplicate_repo: true
+            homepage_url: https://lfnetworking.org/cloud-native-telecom-initiative/
+            logo: logos/CNTi_logo_COLOR.svg
```
Comment (bramwelt) [inline landscape.yml:30]:
```suggestion
            logo: CNTi_logo_COLOR.svg
```

File: landscape.yml [hunk 4]
```diff
@@ -515,6 +507,16 @@ landscape:
             description: >
               The Nephio project is building a Kubernetes-based automation platform for deploying and managing highly distributed, interconnected workloads such
               as 5G Network Functions, and the underlying infrastructure on which those workloads depend.
+          - item:
+            name: Paraglider
+            allow_duplicate_repo: true
+            homepage_url: https://paragliderproject.io
+            logo: logos/paraglider-logo.svg
```
Comment (bramwelt) [inline landscape.yml:514]:
```suggestion
            logo: paraglider-logo.svg
```

**Review by bramwelt — CHANGES_REQUESTED** [review]
Hey @mrsjackson76 , last this is the `logo:` reference should include just the filename: no `logo/` prefix is needed.

**Review by bramwelt — APPROVED** [review]

**Pull request merged**